src/owens_t.jl

```diff
@@ -0,0 +1,63 @@
+"""
+    owens_t(x, a)
+
+Owen's T function
+
+Ported from Matlab implementation of John Burkardt, which is distributed under the GNU LGPL license.
```

Comment:
This is more restrictive than the SpecialFunctions.jl license, which also disqualifies anything derived from that code.